.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.1.0
+    rev: v4.3.0
```

Comment:
Hm, looks like you did `pre-commit update` or something? I *think* you're not supposed to do that; to un-do you can do `git checkout main .pre-commit-config.yaml`.